early-attack-detector-py/src/agent.py

```diff
@@ -262,7 +269,10 @@ def detect_malicious_contract(
     findings = []
 
     if created_contract_address is not None and code is not None:
-        if len(code) > BYTE_CODE_LENGTH_THRESHOLD:
+        from_label_type = "contract" if is_contract(w3, from_) else "eoa"
+        from_eoa_in_tp_list = from_label_type == "eoa" and from_ in TP_ATTACKER_LIST
```

Comment:
I think at this point it's better to only set a  `in_tp_list` variable, and then call `is_contract` at a later stage. Otherwise, we'll be calling it in every contract creation